### DISCUSSION TITLE
Should we add automated breadcrumb block into the meta block area on pages

### DISCUSSION THREAD

**Discussion by carolinejcourtney** [amnestywebsite/humanity-theme#165] [Feature Requests]
https://www.nngroup.com/articles/breadcrumbs/ 

There is a UX issue if sites have a complex hierarchy of Pages but without a breadcrumbs UI that shows users how the page they're reading sits within that hierarchy. Users can add this manually using the 'Back Link' button style, but it is often forgotten or the link is broken.

**Comment by AmnestyAM** [comment]
The original design was designed not to have breadcrumbs in the traditional sense, but we had links back to parent taxonomy terms in the single post UI, and we use a back button on some page layouts, but worth revisiting this question.

## Meta Area
https://www.amnesty.org/en/latest/news/2024/04/a-chance-for-africa-to-counter-the-pitfalls-of-international-criminal-justice/
![image](https://github.com/amnestywebsite/humanity-theme/assets/37809410/13d51c6c-502b-48ce-a778-b6d8e3647b66)

## Back Link with a button block 
https://www.amnesty.org/en/what-we-do/armed-conflict/
![image](https://github.com/amnestywebsite/humanity-theme/assets/37809410/a7bd7b1e-1b21-43b8-92e3-4b5367c57e57)

Would be nice if the block, had the same UI as the meta block, and maybe we could selected to show content such as the share, byline, date etc to keep things consistent . 

We should think about how this will work across posts with meta area, posts without a side bar, pages with header block and without header block too. 

https://github.com/amnestywebsite/amnesty-wp-theme/issues/1238 Old question about breadcrumbs 
https://github.com/amnestywebsite/amnesty-wp-theme/issues/1275 Search Console issue with breadcrumbs 
https://github.com/amnestywebsite/amnesty-wp-theme/issues/2685 Ticket to remove meta box from pages as option did not work
